src/gui/src/components/explorer-grid/selected-item/tabs-insight.tsx

```diff
@@ -113,6 +116,60 @@ const InsightItem = ({
   )
 }
 
+const InsightScore = () => {
+  const { packageScore } = useSelectedItem()
+
+  if (!packageScore) return null
+
+  const scoreOrder = [
+    'overall',
+    'quality',
+    'supplyChain',
+    'maintenance',
+    'vulnerability',
+    'license',
+  ]
+
+  const scores = Object.entries(packageScore)
+    .map(([key, value]) => ({
+      id: key,
+      name: key === 'supplyChain' ? 'supply chain' : key,
+      value: value * 100,
```

Comment:
Same as the other related comment, let's err on the safe side here.

```suggestion
      value: Math.round(value * 100),
```